firefox-ios/Client/Frontend/Browser/Toolbars/Redux/AddressState.swift

```diff
@@ -0,0 +1,141 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Redux
+import ToolbarKit
+
+struct AddressState: StateType, Equatable {
```

Comment:
nit: AddressState and NavigationState are a little ambigious names, adding the word bar to both would probably help or if you can think of a more descriptive name. NavigationState is particularly problematic because of how many different things the word navigation can refer to.